oem/ibm/libpldmresponder/inband_code_update.cpp

```diff
@@ -435,7 +435,9 @@ void CodeUpdate::setVersions()
                     }
                     catch (const sdbusplus::exception::exception& e)
                     {
-                        std::cerr << "Error in getting Activation status \n";
+                        std::cerr << "Error in getting Activation status,"
+                                  << "ERROR=" << e.what() 
```

Comment:
do you also wanna add other information? such as which interface, or object you are accessing that failed.

Reply:
done